modules/machineset-azure-capacity-reservation.adoc

```diff
@@ -0,0 +1,101 @@
+[id="machineset-azure-capacity-reservation_{context}"]
+= Configuring Capacity Reservation by using machine sets
+
+{product-title} version 4.15.25 and later supports on-demand Capacity Reservation with Capacity Reservation groups on {azure-full} clusters.
+
+You can configure a machine set to deploy machines on any available resources that match the parameters of a capacity request that you define.
+These parameters specify the VM size, region, and number of instances that you want to reserve.
+If your {azure-short} subscription quota can accommodate the capacity request, the deployment succeeds.
+
+For more information, including limitations and suggested use cases for this {azure-short} instance type, see the {azure-full} documentation about link:https://learn.microsoft.com/en-us/azure/virtual-machines/capacity-reservation-overview[On-demand Capacity Reservation].
+
+[NOTE]
+====
+You cannot change an existing Capacity Reservation configuration for a machine set.
+To use a different Capacity Reservation group, you must replace the machine set and the machines that the previous machine set deployed.
+====
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You installed the {oc-first}.
+* You created a Capacity Reservation group.
++
+For more information, see the {azure-full} documentation link:https://learn.microsoft.com/en-us/azure/virtual-machines/capacity-reservation-create[Create a Capacity Reservation].
+
+.Procedure
+
+. In a text editor, open the YAML file for an existing machine set or create a new one.
+
+. Edit the following section under the `providerSpec` field:
++
+--
+.Sample configuration
+[source,yaml]
+----
+ifndef::cpmso[]
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+endif::cpmso[]
+ifdef::cpmso[]
+apiVersion: machine.openshift.io/v1
+kind: ControlPlaneMachineSet
+endif::cpmso[]
+# ...
+spec:
+  template:
+ifndef::cpmso[]
+    spec:
+      providerSpec:
+        value:
+          capacityReservationGroupID: <capacity_reservation_group> # <1>
+endif::cpmso[]
+ifdef::cpmso[]
+    machines_v1beta1_machine_openshift_io:
+      spec:
+        providerSpec:
+          value:
+            capacityReservationGroupID: <capacity_reservation_group> # <1>
+endif::cpmso[]
+# ...
+----
+<1> Specify the ID of the Capacity Reservation group that you want the machine set to deploy machines on.
+--
+
+.Verification
+
+* To verify machine deployment, list the machines that the machine set created by running the following command:
++
+[source,terminal]
+----
+ifndef::cpmso[]
+$ oc get machines.machine.openshift.io \
+  -n openshift-machine-api \
+  -l machine.openshift.io/cluster-api-machineset=<machine_set_name>
+endif::cpmso[]
+ifdef::cpmso[]
+$ oc get machine \
+  -n openshift-machine-api \
+  -l machine.openshift.io/cluster-api-machine-role=master
+endif::cpmso[]
+----
+ifndef::cpmso[]
++
+where `<machine_set_name>` is the name of the compute machine set.
```

Comment:
Interesting. I've always seen this handled as a definition list, per the [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks), but I see what you are doing is based off the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=elements-command-line-data-entry) since you have only one user-defined variable.

I'm not sure if this is just the only time I've stumbled across this usage with only one variable, or if I've seen it as a single variable list. Though avoiding a list with a single item is kinda tech comm 101. :)

Reply:
Yeah, there are definitely a few like this throughout the docs. I would love us to be consistent, but that seems off the table for the moment :sweat_smile:

Reply:
This way is definitely "the right way" :tm: 

The joys of peer review!